src/indexer/Matchers.cpp

```diff
@@ -125,8 +125,10 @@ void hdoc::indexer::matchers::FunctionMatcher::run(const clang::ast_matchers::Ma
   for (const auto* i : res->parameters()) {
     hdoc::types::FunctionParam a;
     a.name      = i->getNameAsString();
-    a.type.name = i->getType().getAsString(pp);
-    a.type.id   = getTypeSymbolID(i->getType());
+    auto effectiveType = i->getType();
+    effectiveType.removeLocalFastQualifiers(); // drop top-level cvr-qualifiers since they are not part of the signature
```

Comment:
I'm not sure if `removeLocalFastQualifiers` is the best way to do this, but if it works it works.

Reply:
It probably is a bit brittle, OTOH I can't think of any qualifier we _dont_ want to strip in this position.